plugin/parser/ast/alter_table_stmt.go

```diff
@@ -0,0 +1,9 @@
+package ast
+
+// AlterTableStmt is the struct for alter table statement.
+type AlterTableStmt struct {
+	node
+
+	Table *TableName
+	Cmds  []Node
```

Comment:
Cmd isa  bit cryptic

Reply:
PostgreSQL calls as `action` and MySQL calls as `alter_option`.

I prefer `AlterAction` about this. WDYT. @tianzhou @d-bytebase 

References:
- https://www.postgresql.org/docs/current/sql-altertable.html
- https://dev.mysql.com/doc/refman/8.0/en/alter-table.html